docs/element-definitions.md

```diff
@@ -51,9 +44,11 @@ The Base API Element contains four properties: `element`, `meta`, `attributes`,
         - (string)
         - (number)
         - (boolean)
-        - (array)
-        - (object)
-        - (Base API Element)
+        - (array[Element])
+        - (Element)
+        - (object) - Key Value Pair
```

Comment:
Yeah, I think when we wrote this we didn't define the `Key Value Pair`. This can be updated now.